terra-jupyter-aou/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.10 AS python
+
+FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:0.0.11
+
+# copy everything pip installed from the python image
+COPY --from=python /usr/local/lib/python3.7/dist-packages /usr/local/lib/python3.7/dist-packages
+
+USER root
+
+# need to apt-get everything for python since we can only copy pip installed packages
```

Comment:
is this comment still true?

Reply:
If it's true for the gatk image it should be true here, since that's all we're copying above. @rtitle as to whether it's still true.

Seems likely to me to be true since you can't inherit from two base images

Reply:
Yeah.. we only copy the pip packages from the python image